README.md

```diff
@@ -115,6 +119,8 @@ Returns the `trap`.
 These options are used to override the focus trap's default behavior for this particular activation.
 
 - **onActivate** `{() => void | null | false}`: Default: whatever you chose for `createOptions.onActivate`. `null` or `false` are the equivalent of a `noop`.
+- **onPostActivate** `{(value: unknown) => void | null | false}`: Default: whatever you chose for `createOptions.onPostActivate`. `null` or `false` are the equivalent of a `noop`.
+- **checkCanFocusTrap** `{(containers: Array<HTMLElement | SVGElement | string | () => HTMLElement | SVGElement>) => Promise<unknown> | null | false}`: Default: whatever you chose for `createOptions.checkCanFocusTrap`. `null` or `false` are the equivalent of a `noop`.
```

Comment:
Return type should just be `=> Promise`, and no need for "`null` or `false` are the equivalent of a `noop`." at the end.

Reply:
I made the return type `Promise<void>` since `Promise` on it's own isn't allowed.